client/components/tinymce/plugins/wpcom-view/views/embed/view.jsx

```diff
@@ -19,25 +16,12 @@ import QueryEmbed from 'components/data/query-embed';
 import ResizableIframe from 'components/resizable-iframe';
 
 class EmbedView extends Component {
-	state = {
-		wrapper: null,
-	};
+	view = createRef();
+
+	iframe = createRef();
 
 	componentDidMount() {
-		// Rendering the frame follows a specific set of steps, whereby an
-		// initial rendering pass is made, at which time the frame is rendered
-		// in a second pass, before finally setting the frame markup.
-		//
-		// TODO: Investigate and evaluate whether we need to avoid rendering
-		//       the iframe on the initial render pass
-		// eslint-disable-next-line react/no-did-mount-set-state
-		this.setState(
-			{
-				// eslint-disable-line react/no-did-mount-set-state
-				wrapper: this.refs.view,
-			},
-			this.setHtml
-		);
+		this.setHtml();
```

Comment:
`componentDidMount` should also call `constrainEmbedDimensions()` here.

Previously, there was a `setState( { wrapper } )` call that immediately triggered a second render. On that second render, `componentDidUpdate` was called with in turn called `constrainEmbedDimensions`.

But now, when this `setState()` is gone, there can be just one initial render with no further updates.